core/src/main/java/io/grpc/internal/AbstractClientStream2.java

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright 2014, Google Inc. All rights reserved.
```

Comment:
Although it doesn't seem like it, this was a copy of `AbstractClientStream`, so I didn't change the copyright date.